<!DOCTYPE HTML>
<!--
	Paradigm Shift by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
<link rel="icon" href="favicon.png">
	<head>
		<title>Dr. Steven J. Dunn - Technical Product Manager | Bacterial Genomics</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<meta name="description" content="" />
		<meta name="keywords" content="Bacterial Genomics, Product Management, Technical Product Manager,Researcher, Bioinformatics, Microbial, Genomics, Illumina, MinION, ONT, Figure Design, Scientific, Figures, Consultant" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-70767943-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-70767943-1');
</script>
	</head>
	<body class="is-preload">
  <script src="https://kit.fontawesome.com/19991f4ba5.js" crossorigin="anonymous"></script>

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Intro -->
					<section class="intro">
						<header>
							<h1>Dr. Steven J. Dunn</h1>
							<p>Microbial Genomics</p>
							<p>
								<ul class="icons">
								<li><a href="https://twitter.com/StevenJDunn" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
								<li><a href="https://www.linkedin.com/in/stevenjdunn" class="icon brands fa-linkedin-in"><span class="label">LinkedIn</span></a></li>
								<li><a href="https://github.com/stevenjdunn/" class="icon brands fa-github"><span class="label">GitHub</span></a></li>
								<li><a href="https://github.com/stevenjdunn/stevendunn/blob/main/Dr_Steven_J_Dunn_CV_Resume_Curriculum_Vitae_2021_Redacted.pdf" class="icon solid fa-file-alt"><span class="label">CV/Resume</span></a></li>
							</ul>
							</p>
							<ul class="actions">
								<li><a href="#first" class="arrow scrolly"><span class="label">Next</span></a></li>
							</ul>
						</header>
						<div class="content">
							<br><br><br>
							<p>Hi, I'm Steven. I'm a technical product manager with <strong>10 years of experience</strong> in the microbial genomics field, and a passion for product strategy, design, and  development.</p>
							<p>I also operate a scientific consultancy - Dunn Genomics - that covers bioinformatics, process development and optimisation, medical and scientific writing, data visualisation and business strategy.</p>
							<p>Got a cool dataset? Think I'd be a good fit for a job?</p>
							<strong><a href="https://twitter.com/StevenJDunn">Get in touch!</a></p></strong>


						</div>
					</section>

				<!-- Section -->

				<!-- Section -->
					<section>
						<header>
							<h2>Skills</h2>
						</header>
						<div class="content">
              <p>I'm experienced with biotech and life science product management, with an excellent technical background that underpins my success in realising your business' potential.</p>
							<p>I have a strong background in both wet and dry lab techniques, and extensive experience in large scale CLI based data analyses.</p>
							<p>I've worked on projects characterising a handful of isolates in great depth, to analysing ~9000 genomes, and I have developed production scale WGS Nextflow pipelines executed on AWS Batch.</p>
							<ul>
                <li class="fa-solid fa-wand-magic-sparkles"></i>">Product Management</li>
                <li class="fa-solid fa-laptop">Bioinformatics</li>
                <li class="fa-solid fa-dna">WGS</li>
                <li class="fa-solid fa-braille">RNASeq</li>
                <li class="fa-solid fa-code">Nextflow</li
                <li class="fa-brands fa-aws">AWS</li>
                <li class="fa-brands fa-python">Python</li>
                <li class="fa-solid fa-r">R</li>
                <li class="fa-solid fa-pencil">Visual Design</li>
                <li class="fa-solid fa-briefcase">Business Strategy</li>
                <li class="fa-solid fa-comments-dollar">Marketing</li>

							</ul>
							<p>I also have a strong ability to convey complex information in visually appealing and innovative ways, which has served me well in such an information rich field.</p>
						</div>
					</section>

				<!-- Section -->
					<section>
						<header>
							<h2>Employment</h2>
						</header>
						<div class="content">
							<p>I currently work at a microbial genome sequencing company <a href="https://microbesng.com">MicrobesNG</a> as a Technical Product Manager.</p>
              <p>I'm responsible for the company's full product line, and work with all departments to launch new and exciting services, as well as improve our existing offerings!</p>
							<p> I always throw my self head-first into new projects and roles, and any job that has problem solving and teamwork at its core is a job I would thrive in.</p>

							<!-- Section -->
              <section>
                <header>
                  <h3>Technical Product Manager / New Services Bioinformatician, MicrobesNG</h3>
                  <p>2022 - Present</p>
                </header>
                <div class="content">
                  <p>I joined MicrobesNG in the summer of 2022 as their New Services Bioinformatician. I was responsible for developing their analysis pipelines for new services, and ensuring that those pipelines could operate efficiently in a large-scale production environment.</p>
                  <p>If you've worked in a small business before, you'll know the challenges and excitement that sort of environment can bring. You tend to wear many hats out of necessity, and I quickly found that my diverse skill set could help the business across many more aspects than just their data analysis pipelines.</p>
                  <p>So, in 2023 I was appointed as a Junior Technical Product Manager, responsible for the company's full suite of WGS services. </p>
                  <p>I have focussed on improving the quality and accessibility of the company's offerings, and ensuring that their products align with the company's objectives, culture and core values. This included a service-suite level rebrand, as well as the integration of new products that sit in their own service line up.</p>
                  <p>My strong technical background in this area helps me achieve the best outcome with the limited resources available, and prioritise, strategise and develop products in a meaningful and collaborative way.</p>
                  </div>
                </div>
              </section>
              
								<section>
									<header>
										<h3>Postdoctoral Researcher, University Of Birmingham</h3>
										<p>2017 - 2022</p>
									</header>
									<div class="content">
										<p> My work at the Institute of Microbiology and Infection has mainly focussed on the <strong>evolutionary mechanisms</strong> underpinning AMR dissemination in <strong>MDR <i>E. coli</i></strong>. This BBSRC funded work introduced a CTX-M-15 bearing plasmid to a range of ESBL-naive <i>E. coli</i>, and followed up by looking at the <strong>genomic</strong> and <strong>transcriptomic</strong> impact of that plasmid acquisition.</p>
										<p>You can check out our paper on <a href="https://msystems.asm.org/content/6/2/e00083-21">that here.</a></p>
										<p>We then developed this further by putting these hosts through <strong>600 generations</strong> of experimental growth. We saw a number of <strong>parallel variations</strong> occurring, both in the same gene or position, as well as the same operon or pathway. A lot of variation was driven by IS movement.</p>
										<p>I also had the opportunity to work on many <strong>diverse and enjoyable side projects</strong>, one of the works I am most proud of is a study looking at a cohort of international travellers. We took daily stool samples to investigate the dynamics of intestinal colonisation. And it is <strong><i>dynamic</i></strong>.</p>
										<p>You can find the <a href="https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(20)30224-X/fulltext">paper on that work here</a>, or check out this lightning talk I made for ABPHM21.</p>
										<p><iframe width="560" height="315" src="https://www.youtube.com/embed/7Od11MQCI3A" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
										<p>Of course, I worked alongside many other great postdocs and PhD students, and had the pleasure of helping to supervise some really excellent projects. It's through this often underappreciated aspect of postdoc work that I really found my <strong>affinity for helping others</strong>. I love my job, and a large part of that is being able to bring other people up - teaching them to do something they think is impossible, or showing them a new way to achieve something in a 10th of the time, or just giving them feedback and seeing their confidence and abilities grow. This is a large part of my motivation to work in academia, and something I want to develop further as I continue through my career</p>
										<p>Another highlight of my time at the University of Birmingham was running a <a href="https://twitter.com/hashtag/AMGD2020?src=hashtag_click">week long CPD crash-course</a> on microbial genomic analysis. I designed and delivered the curriculum and materials, including a detailed handbook that took people right from using bash, to generating complex annotated phylogenies. The course was a huge success, receiving very high feedback, and a second round sold out in a few days.
										</div>
									</div>
								</section>

								<section>
									<header>
										<h3>COVID-19 Testing Site Manager, University Of Birmingham</h3>
										<p>2020 - 2021</p>
										<p><strong>AND</strong></p>
										<h3>COVID-19 Laboratory Technician, Medicines Discovery Catapult</h3>
										<p>2020 - 2020</p>
									</header>
									<div class="content">
										<p>The pandemic was a period of uncertainty and strife for all of us, and adjusting to this was a great challenge. Fortunately, I was to be able to offer my experience as a scientist to help the COVID-19 testing efforts, which helped immensely with my own adjustment and mental health during such a strange time.</p>
										<p>This began in the early phases of the pandemic by working as a laboratory technician responsible for the <strong>high-throughput processing</strong> of <a href="https://www.gov.uk/government/publications/coronavirus-covid-19-testing-data-methodology/covid-19-testing-data-methodology-note">Pillar-2</a> PCR swabs. This was based out of Alderley Park, a beautiful research hub based in Macclesfield, and managed by many of the park's excellent staff from companies like Astra Zeneca. Macclesfield is quite the commute from my house, and I would often stay overnight at Manchester Airport, which was eerily deserted.</p>
										<p>As the UK developed its response and validated lateral flow tests for asymptomatic diagnostics, I was assigned to <strong>manage the asymptomatic test site</strong>, housed in The University of Birmingham's Great Hall. This was an incredible feat, orchestrated by many excellent people within the organisation, and allowed us to mass screen our student population before they returned home to their loved ones over the 2020 holiday period.</p>
										<p><iframe width="560" height="315" src="https://www.youtube.com/embed/KP95GbPO_O0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
										<p> We continued offering this testing to all students and staff in 2021, and we also produced a research paper that showed that the tests we were using picked up individuals with high CT-values <a href="https://dx.plos.org/10.1371/journal.pbio.3001216">very reliably.</a></p>
										</div>
								</section>

								<section>
									<header>
										<h3>Hourly Paid Lecturer, Nottingham Trent University</h3>
										<p>2013 - 2017</p>
									</header>
									<div class="content">
										<p>During my PhD, I helped to run the practical undergraduate lab sessions. Halfway through, NTU completed their <i>superlab</i>, which made use of a lot of novel technology for delivering many classes at once.</p>
										<p>It was always great fun, and I had the opportunity to meet and teach a wide variety of undergraduates from many different disciplines.</p>
										</div>
								</section>

								<!-- Section -->
									<section>
										<header>
											<h2>Publications</h2>
											<p> <strong>*</strong> - First Authors
										</header>
										<div class="content">
                      <p>TD Troth, RS McInnes,<strong> SJ. Dunn</strong>, J Mirza, <i>et al.</i> Differences in the gut microbiota between Gurkhas and soldiers of British origin. <i>PLOS ONE </i>(Accepted 2023).</p>
                      <p>EA Cummins; RA Moran; AE Snaith; RJ Hall; CH Connor; <strong>SJ Dunn</strong>; A McNally. Parallel loss of type VI secretion systems in two multi-drug resistant Escherichia coli lineages. <i>Microbial Genomics </i>(Accepted 2023).</p>
                      <p>AE Snaith,<strong> SJ. Dunn</strong>, RA Moran, <i>et al.</i> <a href="https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.001000">The highly diverse plasmid population found in Escherichia coli colonizing travellers to Laos and its role in antimicrobial resistance gene carriage.</a> <i>Microbial Genomics</i> (2023). </p>
                      <p>L Carrilero*, <strong>SJ. Dunn*</strong>, RA Moran, A McNally, MA Brockhurst. <a href="https://journals.asm.org/doi/10.1128/msystems.00713-22">Evolutionary responses to acquiring a multidrug resistance plasmid are dominated by metabolic functions across diverse Escherichia coli lineages.</a> <i>MSystems</i> (2023).</p>
                      <p>RS McInnes, AE Snaith, <strong>SJ. Dunn</strong> et al. A novel resistance reversion mechanism in a vancomycin-variable Enterococcus faecium strain. <i>BioRxiv</i> (2023).</p>
                      <p>EM Darby, VN Bavro,<strong> SJ. Dunn</strong>, A McNally, JMA Blair. <a href="https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000964">RND pumps across the genus Acinetobacter: AdeIJK is the universal efflux pump.</a> <i>Microbial Genomics</i> (2023).</p>
                      <p>Y Liu, S Zhu, L Wei, Y Feng, L Cai, <strong>SJ. Dunn</strong>, A McNally, Z Zong. <a href="https://www.nature.com/articles/s43705-022-00163-y">Arm race among closely-related carbapenem-resistant Klebsiella pneumoniae clones.</a> <i>ISME Communications</i> (2022).</p>
                      <p>V Price, <strong>SJ. Dunn</strong>,RA Moran, J Swindells, A McNally. <a href="https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000841#tab2">Whole-genome sequencing enhances existing pathogen and antimicrobial-resistance surveillance schemes within a neonatal unit.</a> <i>Microbial Genomics</i> (2022).</p>
  										<p>F. Ashford, A. Best, <strong>SJ. Dunn</strong>, <i>et al</i>. <a href="https://journals.asm.org/doi/10.1128/jcm.02408-21">SARS-CoV-2 Testing in the Community: Testing Positive Samples with the TaqMan SARS-CoV-2 Mutation Panel To Find Variants in Real Time.</a> <b><i>Journal of Clinical Microbiology</b></i> (2022).</p>
											<p> <strong>SJ. Dunn*</strong>, L. Carilero*, M. Brockhurst, A. McNally. <a href="https://msystems.asm.org/content/6/2/e00083-21">Limited and Strain-Specific Transcriptional and Growth Responses to Acquisition of a Multidrug Resistance Plasmid in Genetically Diverse <i>Escherichia coli</i> Lineages.</a> <b><i>mSystems</i></b> (2021).</p>
											<p>A. Kantele*, E. Kuenzli*, <strong>SJ. Dunn*</strong>, <i>et al</i>. <a href="https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(20)30224-X/fulltext">Real-time sampling of travelers shows intestinal colonization by multidrug-resistant bacteria to be a dynamic process with multiple transient acquisitions.</a> <b><i>The Lancet Microbe</b></i> (2021).</p>
											<p>J. Ferguson,*, <strong>SJ. Dunn*</strong>, <i>et al</i>. <a href="https://dx.plos.org/10.1371/journal.pbio.3001216">Validation testing to determine the sensitivity of lateral flow testing for asymptomatic SARS-CoV-2 detection in low prevalence settings: testing frequency and public health messaging is key.</a> <b><i>PLOS Biology</b></i> (2021).</p>
											<p>D. Papakonstantinou*, <strong>SJ. Dunn</strong>, <i>et al</i>. <a href="https://journals.asm.org/doi/full/10.1128/mSphere.01224-20">Mapping Gene-by-Gene Single-Nucleotide Variation in 8,535 <i>Mycobacterium tuberculosis</i> Genomes: a Resource To Support Potential Vaccine and Drug Development.</a> <b><i>mSphere</b></i> (2021).</p>
											<p><strong>SJ. Dunn*</strong>, C Connor, A Mcnally. <a href="https://www.sciencedirect.com/science/article/pii/S1369527419300025?via%3Dihub">The evolution and transmission of multi-drug resistant <i>Escherichia coli and Klebsiella pneumoniae: the complexity of clones and plasmids.</a> <b><i>Current Opinion in Microbiology</b></i> (2019).</p>
											<p><strong>SJ. Dunn*</strong>, B Pascoe, J Turton, <i>et al</i>. <a href="https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000227" >Genomic epidemiology of clinical Campylobacter spp. at a single health trust site.</a> <b><i>Microbial Genomics</b></i> (2018).</p>
											<p>A. Mcnally, Y. Oren, D. Kelly, B. Pascoe, <strong>SJ. Dunn</strong>, <i>et al</i>. <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006280"> Combined Analysis of Variation in Core, Accessory and Regulatory Genome Regions Provides a Super-Resolution View into the Evolution of Bacterial Populations.</a> <b><i>PLoS Genetics</b></i> (2016).</p>
											<p>A. Baig, A. Mcnally, <strong>SJ. Dunn</strong>, <i>et al</i>. <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-2087-y"> Genetic import and phenotype specific alleles associated with hyper-invasion in Campylobacter jejuni.</a> <b><i>BMC Genomics</b></i> (2015).
										</div>
										</div>
									</section>


				<!-- Copyright -->
					<div class="copyright">&copy; Dr. Steven J. Dunn. All rights reserved. Design: <a href="https://html5up.net">HTML5 UP</a>.</div>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
